packages/modules/viewer-react/src/components/BaseViewer.tsx

```diff
@@ -29,9 +29,10 @@ export const BaseViewer = ({
   });
 
   const accessToken = useAccessToken();
+  const isBlankConnection = loaderProps.extents && loaderProps.location && !loaderProps.iTwinId;
```

Comment:
so this will work, but wondering if theres a better way

how would users know when they supply an itwinid that they need to provide auth?
can there be some type magic done for this?

Reply:
mind updating the readme while youre at this?